aurora/radiation.py

```diff
@@ -717,12 +722,11 @@ def read_adf15(path, order=1):
         PEC = np.reshape(PEC, (num_den, num_temp))
 
         # interpolate PEC on log dens,temp scales
+         # interpolation of log10(PEC) to avoid issues at low ne or Te
         pec_fun = RectBivariateSpline(
             np.log10(dens),
             np.log10(temp),
-            np.log10(
-                PEC
-            ),  # interpolation of log10(PEC) to avoid issues at low ne or Te
+            np.log10( PEC), 
```

Comment:
@odstrcilt why eliminating the comment (and adding an unnecessary space)?

Reply:
I have just moved the comment before the function to avoid splitting one line in three.